kalsmic/iReporterApi
# E4218351 integrate edit location incident

**This pull request implements edit the location of a red-flag or intervention record functionality:-**
Logic implemented
- Only a non-admin user can edit the location of a record
- An admin user has no access to this functionality  The button is hidden automatically.
- To edit the location of the record, click on view details, while on the details page, click on the edit location button

**user story** https://www.pivotaltracker.com/epic/show/4218351

**How to test**
- log in  as an admin user and view details of a red-flag or interventions, you will notice there is no update location button
- log in as a normal user and view details of a red-flag or interventions. Try editing the location of a red-flag or intervention record.
- The update location button is hidden for records whose status is not draft

**Links**
- UI :  https://kalsmic.github.io/iReporterApi/UI/index.html

**Credentials**
- Username: `arthur2` Password: `Password123`
- Username: `admin` Password: `iReporter123`
